### PR TITLE
FirebaseInstallations changelog

### PR DESCRIPTION
**Pull request by maksymmalyhin**

### CODE REVIEW

File: FirebaseInstallations/API_KEY_RESTRICTIONS.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+The following SDKs updates introduce a dependency on the [Firebase Installations API](https://pantheon.corp.google.com/apis/library/firebaseinstallations.googleapis.com), a new infrastructure service for Firebase:
+
+- Analytics,
+- Cloud Messaging,
+- Remote Config,
+- In-App Messaging,
+- A/B Testing,
+- Performance Monitoring,
+- ML Kit, and
```
Comment (paulb777) [inline FirebaseInstallations/API_KEY_RESTRICTIONS.md]: No "and"  or commas in a list

**Comment by maksymmalyhin** [comment]
@morganchen12 Looks like we need to add some configuration to the Danger bot [this](https://github.com/firebase/firebase-ios-sdk/pull/4617#issuecomment-571803864) looks a bit confusing.

**Comment by maksymmalyhin** [comment]
@paulb777 The note was composed based on b/145957739, though I agree, I would like to see more details myself. 

@andirayo do you think we can provide more details?

File: FirebaseInstallations/CHANGELOG.md [hunk 2]
```diff
@@ -0,0 +1,4 @@
+# 1.0.0
+
+- [added] The Firebase Installations Service is an infrastructure service for Firebase services that creates unique identifiers and authentication tokens for Firebase clients (called "Firebase Installations") enabling Firebase Targeting, i.e. interoperation between Firebase services.
+- [added] The Firebase Installations SDK introduces the Firebase Installations API. Developers that use API-restrictions for their API-Key may experience blocked requests (https://stackoverflow.com/questions/58495985/). This problem can be mitigated by following the instructions found [here](API_KEY_RESTRICTIONS.md).
```
Comment (morganchen12) [inline FirebaseInstallations/CHANGELOG.md]: Note that these links will point to StackOverflow and GitHub in devsite as well.

**Comment by morganchen12** [comment]
Looks like [Danger's current config](https://github.com/firebase/firebase-ios-sdk/blob/master/Dangerfile#L20) only considers changes to CHANGELOG files, but it should consider additions as well.

File: FirebaseInstallations/CHANGELOG.md (hunk 2 above)
Comment (maksymmalyhin) [reply FirebaseInstallations/CHANGELOG.md]: Thank you for the note. It is the intention so far. If we make a page on the devsite we will update the links.

**Comment by maksymmalyhin** [comment]
I am going to merge the PR and proceed with PRs for changelog entries for other IID dependent SDKs. Later on `API_KEY_RESTRICTIONS.md` will be updated with more details on API Key restrictions.

**Pull request merged**